bin/describe/describe_pp.ml

```diff
@@ -42,11 +42,7 @@ let get_pped_file super_context file =
   let in_build_dir file =
     file |> Path.to_string |> Path.Build.relative context.build_dir
   in
-  let file_in_build_dir =
-    if String.is_empty file then
-      User_error.raise [ Pp.textf "no file is given" ]
-    else Path.of_string file |> in_build_dir |> Path.build
```

Comment:
This string is never empty since it is parsed by cmdliner.

Reply:
Even when provided with quotes? E.g. `""`

Reply:
Actually I'm not sure, I'll drop these changes.